include/ppx/command_line_parser.h

```diff
@@ -164,6 +167,26 @@ class CommandLineParser
 
 private:
     CliOptions        mOpts;
+#if defined(PPX_BUILD_XR)
```

Comment:
I think this and some other code that uses the new flag should also check for applications settings xr.enable, otherwise if the support for XR is compiled in, it just advertises the flag but does nothing with it. 
I don't know if there is a good way to do this now, perhaps after we refactor Application it will make this cleaner.

Reply:
Yeah, I was thinking about that, too. There really isn't a good way to propagate that information at this stage of analysis. But at least the new flag won't interfere with anything in non-XR apps.